java-checks/src/main/java/org/sonar/java/checks/UselessConditionCheck.java

```diff
@@ -46,6 +47,11 @@
 public class UselessConditionCheck extends SubscriptionBaseVisitor {
 
   @Override
+  public void scanFile(JavaFileScannerContext context) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)